indico_zodbimport/modules/event_timetable.py

```diff
@@ -506,6 +530,75 @@ def _migrate_contribution(self, old_contrib, friendly_id):
         contrib._last_friendly_subcontribution_id = len(contrib.subcontributions)
         return contrib
 
+    def _migrate_abstract(self, old_abstract):
+        description = getattr(old_abstract, '_fields', {}).get('content', '')
+        description = convert_to_unicode(getattr(description, 'value', description))  # str or AbstractFieldContent
+        old_contribution = getattr(old_abstract, '_contribution', None)
+        type_ = old_abstract._contribTypes[0]
+        try:
+            type_id = self.legacy_contribution_type_map[type_].id if type_ else None
+        except KeyError:
+            self.importer.print_warning(cformat('%{yellow!}Invalid contrib type {}')
+                                        .format(convert_to_unicode(type_._name)),
+                                        event_id=self.event.id)
+            type_id = None
+
+        accepted_type_id = None
+        accepted_track_id = None
+
+        if old_contribution:
+            assert old_contribution.__class__.__name__ == 'AcceptedContribution'
+            if old_abstract._currentStatus.__class__.__name__ == 'AbstractStatusAccepted':
+                old_contrib_type = old_abstract._currentStatus._contribType
+                if old_contrib_type:
+                    accepted_type_id = self.legacy_contribution_type_map[old_contrib_type].id
+                accepted_track = old_abstract._currentStatus._track
+                accepted_track_id = int(accepted_track.id) if accepted_track else None
+
+        abstract = Abstract(event_new=self.event, legacy_id=old_abstract._id,
```

Comment:
if you end up adding `friendly_id` to abstract set it explicitly here; otherwise it'll cause extra queries